examples/query-complexity/index.ts

```diff
@@ -0,0 +1,46 @@
+import "reflect-metadata";
+import { GraphQLServer, Options } from "graphql-yoga";
+import { buildSchema } from "../../src";
+import queryComplexity from "graphql-query-complexity";
+import { RecipeResolver } from "./recipe-resolver";
+
+async function bootstrap() {
+  // build TypeGraphQL executable schema
+  const schema = await buildSchema({
+    resolvers: [RecipeResolver],
+  });
+
+  // Create GraphQL server
+  const server = new GraphQLServer({ schema });
+
+  // Configure server options
+  const serverOptions: Options = {
+    port: 4000,
+    endpoint: "/graphql",
+    playground: "/playground",
+    validationRules: req => [
+      /**
+       * This  provides GraphQL query analysis to reject complex queries to your GraphQL server.
+       * This can be used to protect your GraphQL servers
+       * against resource exhaustion and DoS attacks.
+       * More doucmentation can be found (here)[https://github.com/ivome/graphql-query-complexity]
+       */
+      queryComplexity({
+        maximumComplexity: 8,
+        variables: req.query.variables,
+        onComplete: (complexity: number) => {
+          console.log("Query Complexity:", complexity);
```

Comment:
We can also react on complexity calculation and do something (@abhikmitra I don't know why we show example with console.log btw 😆)

Reply:
Sure what do you have in your mind other than console log ? I just followed the same format as what is followed in graphql-query-complexity reo

Reply:
I think here you can subtract the amount from user's account points fund.